documentation/writing-policies-configmap-reference.md

```diff
@@ -10,41 +10,84 @@ The Configmap Reference allows the reference of configmap values inside kyverno
 
 To refer Configmap inside any Rule provide the context inside each rule defining the list of configmaps which will be referenced in that Rule.
```

Comment:
To reference values from a ConfigMap inside any Rule define a context inside the rule with one or more ConfigMap  declarations.